modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -342,6 +343,17 @@ protected void validateOperationConfiguration(ConfigurationProvider configuratio
     }
   }
 
+  @Override
+  protected ParameterValueResolver getParameterValueResolver() {
+    return (fieldName) -> {
+      try {
+        return source.getFieldValue(fieldName);
```

Comment:
The `ExtensionMessageSource` doesn't have the logic of how to resolve the field value, is delegating to the `SourceWrapper`. 
I don't see another way to expose the value of a field without exposing the entire concrete source `Class`.